### ISSUE TITLE
`delegate-dom` seems disappeared

### ISSUE THREAD

**Issue by leftstick** [moose-team/friends#164]
Error log as following: 

![snip20160323_4](https://cloud.githubusercontent.com/assets/1059956/13980266/95fd4478-f118-11e5-88da-d32e57966c5c.png)

**Comment by ungoldman** [comment]
This looks to be yet another package affected by the mass unpublishing by @azer today (see https://github.com/azer/left-pad/issues/4).

The package `delegate-dom` was unpublished by @azer and then reregistered by @n-johnson. @n-johnson has for some reason registered hundreds of packages unpublished by @azer then republished them as v2.0.0 with no data, making all of them (at least for now) unusable and unrecoverable.

On top of that, it looks like @azer didn't just unpublish his own packages but all of the packages that he had access to. https://github.com/npm-dom/delegate-dom is a collaborative project (not owned just by @azer) but was unpublished by @azer.

To summarize, both @azer and @n-johnson have created a really bad situation for a lot of people. As a temporary solution we could point to https://github.com/npm-dom/delegate-dom in `package.json`.

If you happen to have a project that @azer or @n-johnson are collaborators on, I would suggest removing them.

**Comment by n-johnson** [comment]
Yikes, that's a bit harsh. I grabbed the packages in order to prevent someone with malicious intentions from grabbing the package and publishing a virus in its place.

I'm more than happy to transfer ownership of any module to its correct owner, I've also been in contact with npm who will be taking back most of the packages later today and replacing with a security warning.

**Comment by ungoldman** [comment]
Thanks @n-johnson. Sorry for the harsh words. I was more than a bit upset late last night as this has affected a lot of projects. I now realize you were trying to do the right thing, it's just not clear when looking at your npm profile.

**Comment by leftstick** [comment]
I trying with [dom-delegate](https://github.com/ftlabs/ftdomdelegate) as replacement, but it requires a little updates in the code as following:

```javascript
delegate.on(el, 'a', 'click', function (e) {
    //.....   
})
```

into 

```javascript
delegate(el).on('click', 'a', function (e) {
    //.....   
})
```

And it works.

Hope you guys find a better way to solve it^^.

**Comment by ungoldman** [comment]
Thanks for pointing this out and showing how you fixed it @leftstick. Your fix was used in #165.
